graylog2-server/pom.xml

```diff
@@ -62,6 +62,14 @@
         <maven.build.timestamp.format>yyyyMMddHHmmss</maven.build.timestamp.format>
         <!-- to work around filtering bug, which makes maven.build.timestamp inaccessible -->
         <build.timestamp>${maven.build.timestamp}</build.timestamp>
+
+        <it.es.httpPort>9200</it.es.httpPort>
```

Comment:
Using the standard port might interfere with an ES instance that is running locally. We should either use a fixed but different port or (even better) use a dynamically assigned port.